src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderMap.cs

```diff
@@ -26,16 +35,35 @@ private CoseHeaderMap(bool isReadOnly)
 
         private ICollection<KeyValuePair<CoseHeaderLabel, CoseHeaderValue>> HeaderParametersAsCollection => _headerParameters;
 
+        /// <summary>
+        /// Gets a collection containing the labels in the <see cref="CoseHeaderMap"/>.
+        /// </summary>
         public ICollection<CoseHeaderLabel> Keys => _headerParameters.Keys;
 
+        /// <summary>
+        /// Gets a collection containing the values in the <see cref="CoseHeaderMap"/>.
+        /// </summary>
         public ICollection<CoseHeaderValue> Values => _headerParameters.Values;
 
+        /// <summary>
+        ///   Gets the number of label/value pairs contained in the <see cref="CoseHeaderMap"/>.
+        /// </summary>
```

Comment:
AFAICT, the current docs style guide say that only constructors and conversion/cast operators should link to the type name.  So I'll stop pointing it out now.